packages/react/src/bags/hooks/__tests__/useBag.test.tsx

```diff
@@ -617,7 +646,7 @@ describe('useBag', () => {
 
         await addItem(
           mockProductId,
-          { quantity: 1, sizeId: 1 },
+          { quantity: 1, sizeId: 1, productAggregatorId: undefined },
```

Comment:
In here, I would suggest to omit the `productAggregatorId` to ensure that we don't provide this property, by default it will assume the `undefined` value

Reply:
Agree, I've updated similar cases as well!